### PR TITLE
Fix node js in readme

### PR DESCRIPTION
**Pull request by artem-malko**
Important fix to get more stars!

### CODE REVIEW

**Comment by artem-malko** [comment]
Come on! Lets merge it!

File: README.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ $('tr').so.filter(':hidden').very.show();
 Dogify works on the bleeding edge of the web.
 
 
-## you can improve your nod js
+## you can improve your nodeJS
```
Comment (lastw) [inline README.md]:
```suggestion
## you can improve your nod js
```

**Review by lastw — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (artem-malko) [reply README.md]: may be `node js`?

Comment (vodolazskikh) [reply README.md]: ноуд жиэсть

**Pull request merged**